README.md

```diff
@@ -29,6 +29,8 @@ Testplane (ex-Hermione) is a battle-hardened framework for testing web apps at a
 
 ⚡ **Infinite Extensibility:** Testplane offers a versatile plugin system with dozens of open-source plugins on GitHub, along with custom reporters, commands, and execution logic.
 
+📦 **Various Test Environments:** With Testplane you can run tests not only in Node.js environment but also in the browser. It means you can run e2e/integration tests in Node.js and unit/component tests in browser.
```

Comment:
here I tried to describe the feature in the style of how Nikolai does it, but I'm not sure that I do it right.

Reply:
> You can run tests not only in Node.js environment but also in the browser

"So what?" the user might ask.

I suggest to write it with component testing title and "because you can run your tests not only in node.js environment, but also in the browser" so the accent is on the CT. Average user wouldn't know the difference between those two and especially couldn't think about "what it is for". He probably will just look at "various test environments" and think about various browsers, or node versions, etc...